lib/api/apiUtils/object/storeObject.js

```diff
@@ -26,7 +26,9 @@ function checkHashMatchMD5(stream, hashedStream, dataRetrievalInfo, log, cb) {
             contentMD5,
         });
         log.trace('contentMD5 does not match, deleting data');
```

Comment:
I meant just one log error, because those two log lines seem to duplicate info (e.g. you may add the 'deleting data' bit to the first log line and make it an error). Thanks!